packages/melody-streams/__tests__/mergeObjectSpec.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 trivago N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { mergeObject } from '../src';
+import { testWith, next, complete, createSubjects } from './util/testHelpers';
+
+describe('attachEvent', () => {
+    it('should take simple objects and turn into stream', async () => {
+        const spec = {
+            foo: 'bar',
+            biz: 'baz',
+            arr: [1, 2, 3, 4],
+        };
+        testWith(mergeObject(spec), () => {}, []);
```

Comment:
Since the outcomes of these test are quite small, could we inline the `expect(...).toEqual(...)` instead of using `expect` and `toMatchSnapshot` inside `testWith`?

Reply:
This would make reading the tests way easier.